cypress/e2e/solutions.cy.js

```diff
@@ -1,70 +0,0 @@
-/// <reference types="cypress" />
-
-import scores from "../fixtures/postScores.json";
-
-let session_Id;
-let set_one_quizId;
-
-describe("'/solutions' endpoint", () => {
-
-    beforeEach(() => {
-        cy.sessionEndpoint().should((response) => {
-            session_Id = response.body.sessionId
-        }).then(() => {
-            cy.scoresEndpoint(scores, session_Id).should((response) => {
-                set_one_quizId = response.body.quizId;
-            });
-        });
-    });

```

Comment:
The cypress test had this beforeEach setup, but as far as I can tell, it didn't actually have any bearing on these tests in the first place (removing this from the cypress test still allowed the test to pass), so I did not replicate it in the pytest.

Reply:
np